melos.yaml

```diff
@@ -3,6 +3,7 @@ repository: https://github.com/supabase/supabase-flutter
 
 packages:
   - packages/*
+  - packages/supabase_flutter/example
```

Comment:
I tried it locally, and it seemed like just by specifying a version in pubspec.yaml instead of a path was enough, hence we might be able to get rid of this line. Would you be able to confirm and if that's the case, remove this line?

Reply:
You are right, it works without melos as well. I inspired from [flutterfire melos.yaml](https://github.com/firebase/flutterfire/blob/master/melos.yaml). The way it currently is, I think melos would update the version of `supabase_flutter` in the example pubspec when we update `supabase_flutter`. Though I don't know if it occurs in the changelog or similar as well, so I'm fine with deleting it as well and time to time update the version manually.